enforcer/datapath.go

```diff
@@ -74,7 +79,12 @@ func NewDatapathEnforcer(
 
 	if mode == constants.RemoteContainer || mode == constants.LocalServer {
 		// Make conntrack liberal for TCP
-		cmd := exec.Command("sysctl", "-w", "net.netfilter.nf_conntrack_tcp_be_liberal=1")
+
+		sysctlCmd, err := exec.LookPath("sysctl")
+		if err != nil {
+			return nil
```

Comment:
This should return an error.

In general all NewSomething that can fail for a reason or another should return an error